prepare.js

```diff
@@ -1,15 +1,23 @@
-const fs = require('fs');
-const path = require("path");
+import fs from 'fs';
+import path from 'path';
 
-const reportMetaData = require("./src/test/resources/report_meta_data.json");
-const axeAssessedPages = require("./src/test/resources/axe_results.json");
+import { fileURLToPath } from 'url';
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+import { createRequire } from "module";
+const require = createRequire(import.meta.url);
+
+const reportMetaData = require('./src/test/resources/report_meta_data.json');
+const axeAssessedPages = require('./src/test/resources/axe_results.json');
```

Comment:
we could probably read the file as a string rather than requiring it since I don't think we're modifying it as json

Reply:
I have removed the required import in favor of using the `fs` library to read in the JSON but we do need it as we do modify the JSON.